### PR TITLE
EPMEDU-627: Add More flow

### PR DESCRIPTION
**Pull request by igorTimofiejczyk**
https://user-images.githubusercontent.com/10826166/187074607-4eb5d57b-0fd3-4a60-8942-143a89fe3cb2.mp4

### CODE REVIEW

File: animeal/src/Flows/Main/Modules/More/MoreContract.swift [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import UIKit
+
+// MARK: - View
+protocol MoreViewable: AnyObject {
+    func applyActions(_ viewItems: [MoreItemView])
+}
+
+// MARK: - ViewModel
+typealias MoreViewModelProtocol = MoreViewModelLifeCycle
```
Comment (marek-stoma) [inline animeal/src/Flows/Main/Modules/More/MoreContract.swift:9]: also you can use parentheses to make the following format:
typealias MoreViewModelProtocol = (
   MoreViewModelLifeCycle &
   ... &
   ...
)

File: animeal/src/Flows/Main/Modules/More/ViewModel/Mappers/MoreViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        self.mapper = mapper
+        self.model = model
+    }
+
+    // MARK: - Life cycle
+    func load() {
+        let actions = model.fetchActions()
+        onActionsHaveBeenPrepared?(
+            actions.map {
+                mapper.mapActionModel($0)
```
Comment (marek-stoma) [inline animeal/src/Flows/Main/Modules/More/ViewModel/Mappers/MoreViewModel.swift:29]: retain cycle here?

Comment (igorTimofiejczyk) [reply animeal/src/Flows/Main/Modules/More/ViewModel/Mappers/MoreViewModel.swift:29]: as discussed

Comment (marek-stoma) [reply animeal/src/Flows/Main/Modules/More/ViewModel/Mappers/MoreViewModel.swift:29]: yep, possible enhancement (to review faster and simpler): 
let actions = model.fetchActions().map { mapper.mapActionModel($0) }
onActionsHaveBeenPrepared?(actions)

**Review by marek-stoma — APPROVED** [review]

**Pull request merged**